Dockerfile

```diff
@@ -403,6 +403,25 @@ RUN mkdir -p $DOTNET_HOME/bin \
 FROM scratch AS dotnet
 COPY --from=dotnetbuild /opt/dotnet /opt/dotnet
 
+#------------------------------------------------------------------------
```

Comment:
Yes, it's e.g. used as part of the `Jenkinsfile`.

Reply:
So, `Dockerfile-legacy` should be updated in this PR / commit as well ?!

Reply:
Ideally, yes. But going forward I'd like to avoid the need to maintain `Dockerfile-legacy` in addition to `Dockerfile`, and I hope @mnonnenmacher's Docker experiments to be the basis for such a change.

That said, I'm not sure whether it's worth to put effort into maintaining `Dockerfile-legacy` just now, as part of this initial implementation. I'd also be fine either adding Bazel support to `Dockerfile-legacy` later, or replacing the whole Docker stuff with a more streamlined solution, whatever comes first.